algonaut_transaction/src/api_models.rs

```diff
@@ -1,9 +1,9 @@
 use algonaut_core::{
```

Comment:
This module (`api_models`) uses the plural while all the other modules (if I don't miss something) use singular names. I would stick to this second convention and rename it as `api_model` or just `model`.